### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by tparnell8**
Issue #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ client.ChannelMessageRecieved += (s, e) =>
         channel.SendMessage(string.Join(", ", channel.Users.Select(u => u.Nick)));
     else if (e.PrivateMessage.Message.StartsWith(".ban "))
     {
-        if (!channel.Operators.Contains(client.User))
+        if (!channel.Users.Contains(client.User))
         {
             channel.SendMessage("I'm not an op here!");
```
Comment (SirCmpwn) [inline README.md]: This doesn't seem to check if the user is an op, it's misleading. Try `UsersByMode['@']`?

**Comment by SirCmpwn** [comment]
Thanks! Looks good except for one error.

**Comment by tparnell8** [comment]
@SirCmpwn I just committed that change

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**